blockchain/filedao/filedao_v2_manager.go

```diff
@@ -19,24 +26,31 @@ type (
 
 	// FileV2Manager manages collection of v2 files
 	FileV2Manager struct {
-		lock    sync.RWMutex
-		Indices []*fileV2Index
+		lock        sync.RWMutex
+		topIndex    uint64
```

Comment:
it is more natural to handle topIndex here, since new file is created and added to Indices     []*fileV2Index